sklearn/linear_model/_logistic.py

```diff
@@ -843,6 +843,12 @@ def _log_reg_scoring_path(X, y, train, test, pos_class=None, Cs=10,
     test : list of indices
         The indices of the test set.
 
+    scoring : callable
```

Comment:
With this change:

https://github.com/scikit-learn/scikit-learn/blob/6c9d8090fa8fe8932960f3108e410875dd41f820/sklearn/linear_model/_logistic.py#L1003-L1004

is not needed anymore